src/server/middleware/validation/index.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from 'express'
+import { ValidationChain, validationResult } from 'express-validator'
+
+const validateRequest = (checks: Array<ValidationChain>) => {
+  return async (req: Request, res: Response, next: NextFunction) => {
+    await Promise.all(checks.map((check: ValidationChain) => check.run(req)))
+
+    const result = validationResult(req)
+    if (result.isEmpty()) {
+      return next()
+    }
+
+    const errors = result.array()
+    return res.send(errors)
```

Comment:
this validation middleware you added is really well done 👍.
before sending error back,I would just add a 400 status code. what do you think?

Reply:
Yes, I will add the code before sending errors back

Reply:
I removed this control in last commit because I am setting default values when validating/sanitizing request params